File: app/src/main/java/io/github/hidroh/tldroid/Application.java
package io.github.hidroh.tldroid;

import android.graphics.Typeface;

public class Application extends android.app.Application {
    public static Typeface MONOSPACE_TYPEFACE;

    @Override
    public void onCreate() {
        super.onCreate();
        MONOSPACE_TYPEFACE = Typeface.createFromAsset(getAssets(), "RobotoMono-Regular.ttf");
    }
}

File: app/src/main/kotlin/io/github/hidroh/tldroid/Application.kt
package io.github.hidroh.tldroid

import android.content.Intent
import android.graphics.Typeface

class Application : android.app.Application() {
  companion object {
    var MONOSPACE_TYPEFACE: Typeface? = null
  }

  override fun onCreate() {
    super.onCreate()
    MONOSPACE_TYPEFACE = Typeface.createFromAsset(assets, "RobotoMono-Regular.ttf")
    startService(Intent(this, SyncService::class.java)
        .putExtra(SyncService.EXTRA_ASSET_TYPE, SyncService.ASSET_TYPE_INDEX))
    startService(Intent(this, SyncService::class.java)
        .putExtra(SyncService.EXTRA_ASSET_TYPE, SyncService.ASSET_TYPE_ZIP))
  }
}
